docs/app/Examples/elements/Label/Variations/LabelExampleSize.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+import React from 'react'
+import { Label } from 'stardust'
+
+const LabelExampleSize = () => {
+  const sizes = ['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']
```

Comment:
I don't want to expose the private _meta property. It may change or completely go away in the future.